lib/helpers.js

```diff
@@ -98,20 +98,18 @@ exports.validateOptionalName = function (name) {
  * Check if relation has a different name from properties
  * @param {Object} modelDefinition The model which has the relation
  * @param {String} name The user input
- * @param {Function(String|Boolean)} Callback to receive the check result.
+ * @returns {String|Boolean} Return the check result.
```

Comment:
hmm...i might be wrong, but I think the return value is not a promise, it's just a pure String/Boolean value right?